### DISCUSSION TITLE
LLAVA, multimodal models not working

### DISCUSSION THREAD

**Discussion by dishantsingla** [open-webui/open-webui#629] [General]
# Bug Report

## Description

**Bug Summary:**
If I attach image and use multimodal model LLaVa, screen is stuck on sending message and nothing happens after that. Ollama is working fine and other models are also working

**Steps to Reproduce:**
- Select LLava as model.
- Attach image. Ask to describe the image

**Expected Behavior:**
Get model response

**Actual Behavior:**
Stuck on chat screen. Nothing happens

## Environment

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I have reviewed the troubleshooting.md document.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
<img width="994" alt="image" src="https://github.com/ollama-webui/ollama-webui/assets/135211326/c005f3f1-88ff-4d5b-b596-78164563713e">

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
<img width="535" alt="image" src="https://github.com/ollama-webui/ollama-webui/assets/25473318/9fa48642-d3de-4100-93bb-a1b0f21e3464">


Unable to reproduce the issue, could you share your browser console log with us? Thanks! Also, please provide all the information listed in the template so that we can better assist you with the troubleshooting.

**Comment by slyt** [comment]
This seems to be related to the input image dimensions. I was able to get it to work with image size of 500x500, 501x500, and 1000x1000. Image size like 4550x2685, 1920x1080 causes the UI to hang.

**Comment by LiangChang2679** [comment]
I have the same problem, how can I solve it? Thanks
